build.fsx

```diff
@@ -141,26 +141,11 @@ Target "SourceLink" (fun _ ->
 // Build a NuGet package
 
 Target "NuGet" (fun _ ->
-    let fsiVersion = GetPackageVersion "packages" "FSharp.Compiler.Service"
-    let scriptcsVersion = GetPackageVersion "packages" "ScriptCs.Hosting"
-    NuGet (fun p -> 
+    Paket.Pack (fun p ->
         { p with   
-            Authors = authors
-            Project = project
-            Summary = summary
-            Description = description
```

Comment:
You can also supply them as assembly attributes in the project being packed if you're packing a single project: see http://fsprojects.github.io/Paket/template-files.html

Having said that, it does assume that you are using the presence of the paket.template file to specify what you are wanting to package; is there a reason that having them built into build.fsx is preferable in some way? It feels a bit like putting your paket.dependencies list in build.fsx to me, but it's entirely possible I'm just not seeing it :)